### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by vallieres**
We are going through the list of codeowners errors and reassigning codeowners to teams identified by execs that have gone through the list.

### CODE REVIEW

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: jekyll-assets
+  annotations:
+    jira/project-key: MKTG
```
Comment (j2em6h) [inline catalog-info.yaml]:
```suggestion
    jira/project-key: WEB
```

File: catalog-info.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+spec:
+  type: service
+  lifecycle: production
+  owner: marketing
```
Comment (j2em6h) [inline catalog-info.yaml]: I'm not familiar with what this is used for, but "marketing" feels overly broad to me? The more specific team would be "web and content"

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1 @@
+*    @circleci/mktg-web-optimizations
```
Comment (j2em6h) [inline .github/CODEOWNERS:1]: This team name may be changing in the near future, but we can update that when/if that happens.

**Review by j2em6h — CHANGES_REQUESTED** [review]

**Comment by j2em6h** [comment]
Additionally, we're only using the `rack-version` branch, which is built off the 2.4 version, not `master`. We don't utilize pull requests or merges, we just reference specific commits, see where this is used here: https://github.com/circleci/circleci.com/blob/main/Gemfile#L19

File: catalog-info.yaml (hunk 2 above)
Comment (vallieres) [reply catalog-info.yaml]: The team is from this list: https://backstage.tools.circleci.com/catalog?filters%5Bkind%5D=group&filters%5Buser%5D=all
Let me know which one it should be!

File: .github/CODEOWNERS (hunk 3 above)
Comment (vallieres) [reply .github/CODEOWNERS:1]: Yes, I'm constantly trying to keep this up to date so let's cross the bridge when we get to it 🙂

**Comment by vallieres** [comment]
> Additionally, we're only using the `rack-version` branch, which is built off the 2.4 version, not `master`. We don't utilize pull requests or merges, we just reference specific commits, see where this is used here: https://github.com/circleci/circleci.com/blob/main/Gemfile#L19

The idea here is to have the right values in Master so that our security tools can correctly assign the Jira to the right team. 🙂  it's all good if you don't use it, it's mainly for our tooling 😄

File: catalog-info.yaml (hunk 2 above)
Comment (j2em6h) [reply catalog-info.yaml]: Thanks @vallieres 

Can there be multiple teams listed here?
I think the best fit is [`web-development-optimization`](https://backstage.tools.circleci.com/catalog/default/group/web-development-optimization)
That team recently got combined with [`content`](https://backstage.tools.circleci.com/catalog/default/group/content), though this doesn't appear to be reflected in that list yet.

Comment (vallieres) [reply catalog-info.yaml]: I think it can only be one. I will that updated with `web-development-optimization`.

**Comment by vallieres** [comment]
Updated and ready for a review 😄 
ping @j2em6h

**Review by j2em6h — APPROVED** [review]

**Pull request merged**